plugins/error-augmentor/plugin.go

```diff
@@ -20,13 +20,16 @@ import (
 	buildeventstream "aspect.build/cli/bazel/buildeventstream/proto"
 	"aspect.build/cli/pkg/ioutils"
 	"aspect.build/cli/pkg/plugin/sdk/v1alpha2/config"
+	"aspect.build/cli/pkg/plugin/sdk/v1alpha2/plugin"
 )
 
 func main() {
 	goplugin.Serve(config.NewConfigFor(NewDefaultPlugin()))
 }
 
 type ErrorAugmentorPlugin struct {
+	plugin.Base
```

Comment:
Will the name of of this be `plugin` or `base`?

Reply:
I don't understand the question.

Reply:
Basically I don't understand (or remember) what this does :P

Reply:
It adds the fields from `plugin.Base` to `ErrorAugmentorPlugin` so that it doesnt need to implement all of the methods. Ive added it here because there is now a `CustomCommands` method that this plugin does not implement